src/script/main/app.ts

```diff
@@ -476,66 +478,47 @@ export class App {
     }
   }
 
-  private _appInitFailure(error: BaseError, isReload: boolean) {
+  private _appInitFailure(error: BaseError) {
+    const {message, type} = error;
     let logMessage = `Could not initialize app version '${Environment.version(false)}'`;
 
     if (Runtime.isDesktopApp()) {
       logMessage += ` - Electron '${platform.os?.family}' '${Environment.version()}'`;
     }
 
-    this.logger.warn(`${logMessage}: ${error.message}`, {error});
+    this.logger.warn(`${logMessage}: ${message}`, {error});
 
-    const {message, type} = error;
     const isAuthError = error instanceof AuthError;
     if (isAuthError) {
       const isTypeMultipleTabs = type === AuthError.TYPE.MULTIPLE_TABS;
       const signOutReason = isTypeMultipleTabs ? SIGN_OUT_REASON.MULTIPLE_TABS : SIGN_OUT_REASON.INDEXED_DB;
       return this.redirectToLogin(signOutReason);
     }
 
-    this.logger.debug(
-      `App reload: '${isReload}', Document referrer: '${document.referrer}', Location: '${window.location.href}'`,
-    );
-    if (isReload) {
-      const isSessionExpired = [AccessTokenError.TYPE.REQUEST_FORBIDDEN, AccessTokenError.TYPE.NOT_FOUND_IN_CACHE];
-
-      if (isSessionExpired.includes(type as ACCESS_TOKEN_ERROR_TYPE)) {
-        this.logger.warn(`Session expired on page reload: ${message}`, error);
-        return this.redirectToLogin(SIGN_OUT_REASON.SESSION_EXPIRED);
-      }
-
-      const isAccessTokenError = error instanceof AccessTokenError;
-      const isInvalidClient = type === ClientError.TYPE.NO_VALID_CLIENT;
-
-      if (isInvalidClient) {
-        return this.redirectToLogin(SIGN_OUT_REASON.SESSION_EXPIRED);
-      }
-
-      if (isAccessTokenError) {
-        this.logger.warn('Connectivity issues. Trigger reload.', error);
-        return window.location.reload();
-      }
```

Comment:
The only difference when it's a reload is that the reason for redirecting to `/auth` is different. 
This doesn't really make sense to me right now, but well as I don't really know what is the intent, I prefer to transcribe as is